packages/web-components/fast-foundation/src/select/select.ts

```diff
@@ -232,36 +210,55 @@ export class FASTSelect extends FormAssociatedSelect {
     public listboxId: string = uniqueId("listbox-");
 
     /**
-     * Calculate and apply listbox positioning based on available viewport space.
+     * Cleanup function for the listbox positioner.
      *
      * @public
      */
-    public setPositioning(): void {
-        const currentBox = this.getBoundingClientRect();
-        const viewportHeight = window.innerHeight;
-        const availableBottom = viewportHeight - currentBox.bottom;
-
-        this.position = this.forcedPosition
-            ? this.positionAttribute
-            : currentBox.top > availableBottom
-            ? SelectPosition.above
-            : SelectPosition.below;
-
-        this.positionAttribute = this.forcedPosition
-            ? this.positionAttribute
-            : this.position;
-
-        this.maxHeight =
-            this.position === SelectPosition.above ? ~~currentBox.top : ~~availableBottom;
-    }
+    public cleanup: () => void;
 
     /**
-     * The max height for the listbox when opened.
+     * Calculate and apply listbox positioning based on available viewport space.
      *
-     * @internal
+     * @public
      */
-    @observable
-    public maxHeight: number = 0;
+    public setPositioning(): void {
+        if (this.$fastController.isConnected) {
```

Comment:
Bit of duplication here with the combo. Is it worth factoring out a shared function?